php/Dockerfile

```diff
@@ -69,9 +69,11 @@ RUN mkdir -p /var/www && \
     chmod +x /usr/local/bin/init-drupal
 
 RUN cd /usr/local/src && \
-    git clone https://github.com/contentacms/contenta_jsonapi && \    
+    git clone https://github.com/contentacms/contenta_jsonapi && \
     cd /usr/local/src/contenta_jsonapi && \
-    git checkout 836683de603741e9e3beebac1b047e34c684a69c && \
+    git fetch --tags  && \
+    latestTag=$(git describe --tags `git rev-list --tags --max-count=1`) # Get latest tag name && \
```

Comment:
I think the comment (including the `#`) is involuntarily commenting `&& \` at the end of the line. Can we move it to a new line instead?